app/models/concerns/itemizable.rb

```diff
@@ -125,7 +125,7 @@ def line_items_attributes(params)
   end
 
   def line_items_quantity_is_at_least(threshold)
-    return if storage_location.nil?
+    return if self.class != Kit && storage_location.nil?
```

Comment:
I'd rather not hardcode this... how about 

`return if respond_to?(:storage_location) && storage_location.nil?`